pilot/pkg/security/trustdomain/bundle.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trustdomain
+
+import (
+	"fmt"
+	"strings"
+
+	"istio.io/pkg/log"
+)
+
+type TrustDomainBundle struct {
+	// The trust domain corresponds to the trust root of a system.
+	// Refer to [SPIFFE-ID](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain)
+	TrustDomain string
+	// The trust domain aliases represent the aliases of `trust_domain`.
+	// For example, if we have
+	// ```yaml
```

Comment:
Is the yaml notation actually working anywhere?

Reply:
Probably not. I copied it from the mesh config. Removed.